helm-charts/secret-store/templates/secretstore.yaml

```diff
@@ -25,9 +25,8 @@ spec:
             {
               "POSTGRES_USER": "postgres",
               "POSTGRES_PASSWORD": "123456789",
-              "SEP6_SECRET": "SAJW2O2NH5QMMVWYAN352OEXS2RUY675A2HPK5HEG2FRR2NXPYA4OLYN",
+              "SECRET": "SAJW2O2NH5QMMVWYAN352OEXS2RUY675A2HPK5HEG2FRR2NXPYA4OLYN",
```

Comment:
`SECRET` is too general. Can we rename the reference server's `AppConfig::secret` to something more meaningful since it is shared between SEPs. 

ex: `paymentSigningSecret`, etc.